admin/views/pupil-register/pupils-list.ejs

```diff
@@ -14,7 +14,7 @@
       <% } %>
       <%- partial('partials/_readonly', {availabilityData}) %>
       <h1 class="govuk-heading-xl">Pupil register</h1>
-      <%- partial('partials/_session_expiry') %>
+      <%- partial('partials/_gds_session_expiry') %>
```

Comment:
Why is it called _gds_session_expiry?  This is not vendored code.

Reply:
@jon-shipley it's the prefix I am using for any component that is using the new toolkit markup and styles. Happy to use another convention, any suggestions?

Reply:
_mtc works for me

Reply:
@jon-shipley I will amend all the component clones that have received new markup and style on the next PR as agreed.